terms.html

```diff
@@ -91,12 +100,23 @@
 specifications. This specification decouples the <a>identity provider</a>
 concept into two distinct concepts: the <a>issuer</a> and the <a>holder</a>.
   </dd>
-  <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
+  <dt><dfn data-lt="issuers|issuer's|issuer(s)|issuing|issued">issuer</dfn></dt>
+  <dd>
+A role a <a>party</a> can perform as it constructs a <a>verifiable credential</a> 
+from a set of <a>claims</a> it has <a>authored</a>, meta-data, and proofs,
+typically for the purpose of transferring that to a <a>holder</a>.
+  </dd>
+  <dt><dfn data-lt="party|parties|party's">party</dfn></dt>
   <dd>
-A role an <a>entity</a> can perform by asserting <a>claims</a> about one or
-more <a>subjects</a>, creating a <a>verifiable credential</a> from these
-<a>claims</a>, and transmitting the <a>verifiable credential</a> to a
-<a>holder</a>.
+An <a>entity</a> that has 'a mind of its own', which means, amongst other things, that it autonomously (sovereignly)
+<ul>
+  <li>sets the objectives it aims to pursue</li>
+  <li>creates and maintains (<a>authors</a>) data, including <a>identifiers</a>, <a>claims</a>, and <a>credentials</a></li>
+  <li>decides on the semantics of all data that it <a>authors</a> (i.e., what the data means)</li>
+  <li>gathers, stores, processes, and disseminates data</li>
+  <li>manages its own logic, deciding what is, and what is not, true, valid, trustworthy, what to reuse (e.g., in terms of shared vocabularies and contexts that other <a>parties</a> have produced), etc.
+</ol>
+Typical examples include people and organizations.
```

Comment:
What about IoT devices? They are capable of issuing VCs as well, how do they fit into this model?

Reply:
IoT devices are capable of actually performing the act of constructing verifiable credentials from some given data, creating the required proofs (and transmitting them).
IoT devices are not parties as they have no 'mind of their own'. They act on behalf of a party (typically some organization) that has the information (in their 'mind')
from which the VC is constructed, and to which the signing keys belong that the IoT device wields. The eSSIF-Lab has a [mental model](https://essif-lab.github.io/framework/docs/terms/pattern-party-actor-action) that describes how this works, but diving into that is outside the scope of this PR.